sortinghat/core/models.py

```diff
@@ -143,6 +145,18 @@ def __str__(self):
         return self.name
 
 
+class Group(MP_Node, EntityBase):
```

Comment:
Thought we were going to name this class `OrganizationGroup` or something similar. Although if it's no mandatory to assing it to an organization, probably it doesn't make sense. Then, we should look for a different name for the table.

Reply:
What do you suggest? OrganizationGroup doesnt make much sense since we discussed that some group don't have to be linked to organizations.

Reply:
To be honest, I'm more towards `Team` to be honest. Many people know about GitHub teams so it's easy to understand what it is.

Reply:
Works for me! I had an initial impression that Team might only feel like small groups (I don't know much about Github teams). However, if you think it will work, we can make the model into Team and call the table 'teams'